projects/transformers/trainer_mixins/distillation.py

```diff
@@ -41,22 +42,14 @@ class DistillationTrainerMixin:
     fall back to regular loss computation.
     """
 
-    def __init__(
-        self,
-        teacher_models=None,
-        kd_ensemble_weights=None,
-        kd_factor_init=1.0,
-        kd_factor_end=1.0,
-        kd_temperature_init=1.0,
-        kd_temperature_end=1.0,
-        **kwargs,
-    ):
+    def __init__(self, *args, **kwargs):
         """
-        Add following variables to Trainer
+        Add following variables under 'mixin_args' through the training arguments.
```

Comment:
Should be trainer_mixin. We might still need model_mixins (or maybe even mixins to the tokenizer class, for example)

Reply:
Fair point, just changed it.